src/constants/Api/Users/api.d.ts

```diff
@@ -1,7 +1,7 @@
 import { IUser } from 'src/redux/users/type'
 
 export type FormData = {
-  id: string
+  id?: string
```

Comment:
are you sure that there are users without id, it is strange, every user should have id field